public/jsx/app.jsx

```diff
@@ -23,7 +24,10 @@ var routes = (
   <Route handler={App}>
     <Route name="login" path="/" handler={Login}/>
     <Route name="card-form" path="/payment/card/" handler={CardDetails}/>
-    <Route name="complete" path="/payment/complete/" handler={CompletePayment}/>
+    <Route
+      name="complete" path="/payment/complete/" handler={CompletePayment}/>
+    <Route
+      name="card-listing" path="/payment/card-list/" handler={CardListing}/>
```

Comment:
but the URLs should be easy to change later since nothing will rely on the URLs.